Readme.md

```diff
@@ -1,21 +1,69 @@
-# Lombiq <add project name here>
+# Lombiq PowerShell Analyzers
 
 
 
 ## About
 
-Add a general overview of the project here. Keep or remove the OSOCE note below as necessary.
+Powershell static code analysis via [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) and [our configuration for it](PSScriptAnalyzerSettings.psd1).
 
 Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects!
 
 
 ## Documentation
 
-Add detailed documentation here. If it's a lot of content then create documentation pages under the *Docs* folder and link pages here.
+### Pre-requisites
 
+The *PSScriptAnalyzer* module must be installed. If it's not installed, then running the script in `pwsh` will output no results and running it in `powershell` will throw an exception. Follow the steps [here](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/overview?view=ps-modules#installing-psscriptanalyzer).
+
+Note that if you are usnig this in GitHub Actions, the common images (`windows-latest` and `ubuntu-latest`) already have it so you don't need to install anything.
+
+### Usage
+
+#### PowerShell CLI
+
+Use the script like this to output the analyzer violations with `Write-Error`:
+
+```pwsh
+./Invoke-Analyzer.ps1 -SettingsPath PSScriptAnalyzerSettings.psd1
+```
+
+The `-SettingsPath` can be omitted, in this case the _PSScriptAnalyzerSettings.psd1_ in the same directory as the _Invoke-Analyzer.ps1_ will be used.
+
+#### GitHub Actions
+
+You can invoke it from an _action.yml_ file like this:
+```yaml
+    - name: Lint PowerShell scripts
```

Comment:
Let's settle on analysis vs linting. Since it's called the former everywhere else, use it here too.